cypress/e2e/editPage.spec.ts

```diff
@@ -14,6 +14,8 @@ import {
 // Constants
 const PRIMARY_COLOUR = "rgb(255, 0, 0)"
 const SECONDARY_COLOUR = "rgb(0, 255, 0)"
+const SUCCESSFUL_EDIT_PAGE_TOAST =
```

Comment:
could we check to see if the old text was being referenced anywhere else? (or if this has been done already, mark thi  as closed)

Reply:
It was, changed in https://github.com/isomerpages/isomercms-frontend/pull/1399/commits/6445e6b05f8b56b2c228ec8fa48e72be84375c65, and also modified other references to all draw from the same source

Reply:
i don't have anything else alr, feel free to merge